DXRModules/DeusEx/Classes/DXRFlags.uc

```diff
@@ -959,7 +972,7 @@ function NewGamePlus()
     }
 
     SetGlobalSeed("NewGamePlus");
-    p.CombatDifficulty *= 1.3;
+    p.CombatDifficulty=FClamp(p.CombatDifficulty*1.3,0,15); //Anything over 15 is kind of unreasonably impossible
     NewGamePlusVal(settings.minskill, 1.2, exp);
     NewGamePlusVal(settings.minskill, 1.2, exp);
```

Comment:
why is minskill increased twice? also maybe we should add an optional argument for the max value to the NewGamePlus function?